run_dir/static/js/clone_project.js

```diff
@@ -0,0 +1,84 @@
+/*
+File: clone_project.js
+URL: /static/js/clone_project.js
+Powers /clone_project - template is run_dir/design/clone_project.html
+*/
+
+
+$('#get-project').on('click', function () {
+    let project_id = $('#inputProj').val();
+    $("#projid").text(project_id);
+    $("#projidModal").text(project_id);
```

Comment:
Maybe move this into the return callback? In case of a failed api request, the project_id should not be set on these two fields right?

Reply:
Done!